src/html/shared/hero.html

```diff
@@ -0,0 +1,21 @@
+<div class="c-hero">
+    <div class="o-wrapper">
+        <div class="o-layout">
+            <div class="o-layout__item  u-4/12@tablet  u-push-1/12@tablet">
+                <div class="c-hero__text">
+                    <h1>The shining light of Shopify</h1>
+                    <p>When you make a sale this light will light up and notify you, how exciting watching your sales light
+                        up... let's start selling!
+                    </p>
+                    <button class="c-watch  c-btn">
```

Comment:
I'm not sure if we need a new component just to position an icon on the left of a button. Maybe something like this would be be better.

```
<button class="c-btn  c-btn--icon-left">
   <div class="c-icon  c-icon--small  c-icon--darkbg">{{ sprite("play") }}</div>
   Watch the video
</button>
```